### ISSUE TITLE
Data loss after opening project

### ISSUE THREAD

**Issue by sanderegg** [ITISFoundation/osparc-issues#429]
@newton1985 :
```all my data seems to be gone from my 3 most recent studies```

The user had 3 studies with jupyter-smash services that were emptied:
- project uuid: 4db56892-75e5-11eb-bc6e-02420a0b02df, name: MR:
  - node uuid: 5233be66-f34a-4ba7-a7a2-b802e12e240a, name: EM LF Viewer:
    - from file_meta_data table: file_id: 4db56892-75e5-11eb-bc6e-02420a0b02df/5233be66-f34a-4ba7-a7a2-b802e12e240a/work.zip, last_modified: 2021-03-03 15:03:50.628000+00:00, file_size: 22
- project uuid: 84fea828-75e4-11eb-bc6e-02420a0b02df, name: GF
  - node uuid: 81dd1861-7c80-45b7-bdfb-f70d524565db, name: EM Viewer:
    - from file_meta_data table: file_id: 84fea828-75e4-11eb-bc6e-02420a0b02df/81dd1861-7c80-45b7-bdfb-f70d524565db/work.zip, last_modified: 2021-03-03 15:19:26.478000+00:00, file_size: 22
- project uuid: 151e2ea6-75e5-11eb-bc6e-02420a0b02df, name: GJ
  - node uuid: 459fd13e-417c-4c71-8ce0-2f70039b3ebb, name: EM LF Viewer:
    - file_id: 151e2ea6-75e5-11eb-bc6e-02420a0b02df/459fd13e-417c-4c71-8ce0-2f70039b3ebb/work.zip, last_modified: 2021-03-02 02:25:05.938000+00:00, file_size: 326

This graylog link shows the entries of all jupyter-smash started at the time, and the suspicious error messages:
```WARNING:/docker/state_puller.py:Unexpected error while retrieving state from S3 for /home/jovyan/work. Skipping due to --silent option```
[Graylog entry](https://monitoring.osparc.speag.com/graylog/streams/000000000000000000000001/search?q=container_name%3Ajupyter-smash%2A&rangetype=absolute&streams=000000000000000000000001&from=2021-03-03T14%3A45%3A05.712Z&to=2021-03-03T15%3A02%3A30.896Z)

Actions to be taken:
- restore S3 backup to access **all** the files of these studies
- fix the issue within the jupyter-lab services
- deploy the fix, inform the users
- update the projects using the failed services
- restore the projects

**Comment by pcrespov** [comment]
For critical bugs, we should have an auto-upgrade mechanism. 
E.g. I have a project with v1.1 . As a user I am interested in the features of the service v1. 
All security patches or bug fixes should be automatically applied (perhaps previous confirmation of the user) and those versions "marked" in the docker registry

**Comment by GitHK** [comment]
@pcrespov after pulling the data from the last  backup, these are the results:

1. for node **4db56892-75e5-11eb-bc6e-02420a0b02df** 
    - `4db56892-75e5-11eb-bc6e-02420a0b02df/f70b0396-d46f-40e5-9764-74956bda916a/work.zip` is 2.53 Gb **previous reported node was not found**
1. for node **84fea828-75e4-11eb-bc6e-02420a0b02df**  
    - `84fea828-75e4-11eb-bc6e-02420a0b02df/599f7400-8aba-4e87-b838-b3a5aae217e0/work.zip` is 1.8Gb **previous reported node was not found**
1. for node **151e2ea6-75e5-11eb-bc6e-02420a0b02df** 
     - `151e2ea6-75e5-11eb-bc6e-02420a0b02df/459fd13e-417c-4c71-8ce0-2f70039b3ebb/work.zip` size is 326 bytes **same as before**
     - `151e2ea6-75e5-11eb-bc6e-02420a0b02df/c81cf4cc-8e24-43fc-8f8f-4a3fcbbaf8fa/work.zip` size is 3.52 Gb **unsure what this node is**

**Comment by GitHK** [comment]
@pcrespov @sanderegg  I've managed to reproduce the below issue on my local deployment (which is running for 5 days now)
`WARNING:/docker/state_puller.py:Unexpected error while retrieving state from S3 for /home/jovyan/work. Skipping due to --silent option`

I've removed the try catch and let all the errors go through. The below logs is showing that **storage is unavailable**.
No further entries are being shown in the logs of storage upon access to the service. It's like it stopped receiving requests.

[jlabservice.log](https://github.com/ITISFoundation/osparc-issues/files/6089103/jlabservice.log)
